genetics/src/main/java/binnie/genetics/genetics/GeneTracker.java

```diff
@@ -30,8 +30,7 @@
 	private GameProfile username;
 
 	public GeneTracker(String s, @Nullable GameProfile username) {
-		super(s);
-		discoveredGenes = new ArrayList<>();
+		this(s);
```

Comment:
I think this should be the "main" constructor so that the other constructor calls `this(s, null)`